alderaan/pipeline-scripts/prometheus.groovy

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env groovy
+
+def pipeline_id = env.BUILD_ID
+def node_label = NODE_LABEL.toString()
+def prometheus_test = PROMETHEUS_TEST.toString().toUpperCase()
+def property_file_name = "prometheus.properties"
+
+println "Current pipeline job build id is '${pipeline_id}'"
+
+// run prometheus scale test
+stage ('prometheus_scale_test') {
+	if ( prometheus_test == "TRUE") {
+		currentBuild.result = "SUCCESS"
+		node(node_label) {
+			if (fileExists(property_file_name)) {
+				println "prometheus.properties file exist... deleting it..."
+				sh "rm ${property_file_name}"
+			}
+			sh "wget -O ${property_file_name} ${PROMETHEUS_PROPERTY_FILE}"
+			sh "cat ${property_file_name}"
+			def prometheus_properties = readProperties file: property_file_name
+			def REFRESH_INTERVAL = prometheus_properties['REFRESH_INTERVAL']
+			def CONCURRENCY = prometheus_properties['CONCURRENCY']
+			def GRAPH_PERIOD = prometheus_properties['GRAPH_PERIOD']
+			def DURATION = prometheus_properties['DURATION']
+
+			println "----------USER DEFINED OPTIONS-------------------"
+			println "-------------------------------------------------"
+			println "-------------------------------------------------"
+			println "REFRESH_INTERVAL: '${REFRESH_INTERVAL}'"
+			println "CONCURRENCY: '${CONCURRENCY}'"
+			println "GRAPH_PERIOD: '${GRAPH_PERIOD}'"
+			println "DURATION: '${DURATION}'"
+			println "-------------------------------------------------"
+			println "-------------------------------------------------"
+
+			// copy the parameters file to jump host
+			sh "git clone https://${token}@${repo} ${WORKSPACE}/perf-dept && chmod 600 ${WORKSPACE}/perf-dept/ssh_keys/id_rsa_perf"
+			sh "scp -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i ${WORKSPACE}/perf-dept/ssh_keys/id_rsa_perf ${property_file_name} root@${jump_host}:/root/properties"
+
+			try {
+				prometheus_build = build job: 'SVT_PROMETHEUS_SCALING',
+				parameters: [	[$class: 'StringParameterValue', name: 'NAMESPACE', value: NAMESPACE ],
```

Comment:
All the parameters needed by the prometheus job should be passed by this script. We shouldn't have default values in the prometheus jenkins job.